docs/reference/search/search.asciidoc

```diff
@@ -92,7 +92,7 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=df]
 
 `docvalue_fields`::
 (Optional, string) A comma-separated list of fields to return as the docvalue
-representation of a field for each hit.
+representation of a field for each hit. See <<docvalue-fields>>.
```

Comment:
This isn't closely related to the change, but I saw the opportunity to refactor the field retrieval docs a bit.

Reply:
These look great. I particularly like use "field pattern."